locations/spiders/timpson_group.py

```diff
@@ -0,0 +1,211 @@
+from typing import Iterable
+
+from scrapy.http import FormRequest, Response
+
+from locations.categories import Categories, apply_category
+from locations.hours import DAYS, OpeningHours
+from locations.items import Feature
+from locations.json_blob_spider import JSONBlobSpider
+from locations.pipelines.address_clean_up import merge_address_lines
+from locations.spiders.asda_gb import AsdaGBSpider
+from locations.spiders.bq import BqSpider
+from locations.spiders.homebase_gb_ie import HomebaseGBIESpider
+from locations.spiders.leyland_sdm_gb import LeylandSdmGBSpider
+from locations.spiders.morrisons_gb import MorrisonsGBSpider
+from locations.spiders.robert_dyas_gb import RobertDyasGBSpider
+from locations.spiders.sainsburys import SainsburysSpider
+from locations.spiders.screwfix_gb import ScrewfixGBSpider
+from locations.spiders.tesco_gb import TescoGBSpider
+from locations.spiders.the_range import TheRangeSpider
+from locations.spiders.waitrose import WaitroseSpider
+from locations.spiders.wickes_gb import WickesGBSpider
+
+
+class TimpsonGroupSpider(JSONBlobSpider):
+    name = "timpson_group"
+    allowed_domains = ["www.maxphoto.co.uk"]
+    start_urls = ["https://www.maxphoto.co.uk/storefinder/locator/get/_all_/"]
+    locations_key = "locations"
+    brands = {
+        "arkhive": {
+            # Note: concept store brand (one off) and therefore not likely to
+            # be present in Wikidata or NSI.
+            "brand": "Arkhive",
+            "brand_wikidata": None,
+            "category": Categories.SHOP_PHOTO,
+            "website_template": None,
+        },
+        "barbershop": {
+            # TODO: not currently defined as a separate brand in the API or in
+            # Wikidata or NSI.
+            "brand": "Barbershop",
```

Comment:
This is in [NSI](https://nsi.guide/index.html?t=brands&k=shop&v=hairdresser&tt=q7807658):
```
brand=Timpson
name=Barbershop by Timpson
shop=hairdresser
```